README.md

```diff
@@ -126,6 +126,31 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `expose.loadBalancer.ports.notaryPort` | The service port Notary listens on. Only needed when `notary.enabled` is set to `true`|
 | `expose.loadBalancer.annotations` | The annotations attached to the loadBalancer service | {} |
 | `expose.loadBalancer.sourceRanges` | List of IP address ranges to assign to loadBalancerSourceRanges | [] |
+| **Internal TLS** |
+| `internalTLS.enabled` | Enable the tls for the components (chartmuseum, clair, core, jobservice, portal, registry, trivy) | `false` |
+| `internalTLS.certSource` | Method to provide tls for the components, options is `auto`, `manual`, `secret` | `auto` |
+| `internalTLS.trustCa` | The content of trust ca, only available when `certSrouce` is `manual` |  |
```

Comment:
this trust CA should be the CA that signs the internal certificate correct? could you clarify that in the doc?
Please also elaborate what the secret should look like when the certsource is set to `secret`